theano/sparse/basic.py

```diff
@@ -233,9 +233,7 @@ def conv_op(*inputs):
         out = op(*ipt)
         return oconv(out)
 
-    seed_rng()
-    rng = numpy.random
-    T.verify_grad(conv_op, dpt, 2, rng, *args, **kwargs)
```

Comment:
We need a way to keep the deterministic behavior. So we need to continue to pass a seeded rng and we need to continue to use the mechanism in utt. Instead of duplicating the code here, I see 2 possibles fix:

- In unittest_tools.py put a try..except ImportError around the nose parametrised.
- Move the sparse_verify_grad to a test file. It is only used there. I did that in this PR: https://github.com/Theano/Theano/pull/4944

@abergeron, what do you think?